src/it/java/org/jboss/tools/intellij/quarkus/tests/BasicTest.java

```diff
@@ -115,6 +121,20 @@ private void createQuarkusProject(RemoteRobot remoteRobot, String projectName, B
 
         QuarkusNewProjectFinalPage quarkusNewProjectFinalPage = newProjectDialogWizard.find(QuarkusNewProjectFinalPage.class, Duration.ofSeconds(10));
         quarkusNewProjectFinalPage.setProjectName(projectName);
+
+        String QUARKUS_PROJECT_LOCATION = CreateCloseUtils.PROJECT_LOCATION + File.separator + projectName;
+        Path quarkusProjectDir = Paths.get(QUARKUS_PROJECT_LOCATION);
+        boolean doesProjectDirExists = Files.exists(quarkusProjectDir);
+        if (!doesProjectDirExists) {
+            try {
+                Files.createDirectories(quarkusProjectDir); // create project directory with project name to prevent "Do you want Eclipse to create new directory..." popup
```

Comment:
The comments is about Eclipse, do we have the same problem with Intellij? if yes, please change the comments.

Reply:
Fixed :)